frontend/packages/console-dynamic-plugin-sdk/src/utils/k8s/hooks/useModelsLoaded.ts

```diff
@@ -1,16 +1,15 @@
 import * as React from 'react';
-// eslint-disable-next-line @typescript-eslint/ban-ts-ignore
-// @ts-ignore: FIXME missing exports due to out-of-sync @types/react-redux version
+import { Map as ImmutableMap } from 'immutable';
 import { useSelector } from 'react-redux';
 import { K8sModel } from '../../../api/common-types';
 import { OpenShiftReduxRootState } from './k8s-watch-types';
 
 export const useModelsLoaded = (): boolean => {
   const ref = React.useRef(false);
-  const k8sModels = useSelector<OpenShiftReduxRootState, K8sModel>(({ k8s }) =>
-    k8s.getIn(['RESOURCES', 'models']),
+  const k8sModels = useSelector<OpenShiftReduxRootState, ImmutableMap<string, K8sModel>>(
+    ({ k8s }) => k8s.getIn(['RESOURCES', 'models']),
   );
-  const inFlight = useSelector<OpenShiftReduxRootState, K8sModel>(({ k8s }) =>
+  const inFlight = useSelector<OpenShiftReduxRootState, boolean>(({ k8s }) =>
```

Comment:
:partying_face:

Reply:
Yeah I know. The types were wrong at a lot of places :P